app/models/user.rb

```diff
@@ -0,0 +1,6 @@
+class User < ApplicationRecord
+  devise :database_authenticatable, :registerable, :validatable
+  validates :name, presence: true
```

Comment:
- Please, kindly add the length attribute to control the length of the name so you don't end up having a name that is very long eg. 
- ` validates :name, presence: true, length: { maximum: 50 }`